### PR TITLE
[WIP] Revise the documentation

### PR DESCRIPTION
**Pull request by settermjd**
The intent here is to make it that much easier to read. There were a few typos and grammar mistakes. But the main thing that I focused on was readability. I hope that this helps.

### CODE REVIEW

**Comment by shochdoerfer** [comment]
We probably should add the "Why Disco?" part to the README.md file to give a better impression why Disco as a DI container does make sense.

File: docs/intro/getting-started.md [hunk 1]
```diff
@@ -2,35 +2,27 @@
 
 ## Prerequisites
 
-Disco needs at least [PHP](http://php.net) 7.0 since Disco relies on the
-[return type declarations](http://php.net/manual/en/functions.returning-values.php#functions.returning-values.type-declaration)
-feature introduced with PHP 7.0.
+Disco needs at least [PHP](http://php.net) 7.0, as Disco relies on the [return type declarations](http://php.net/manual/en/functions.returning-values.php#functions.returning-values.type-declaration) feature introduced with PHP 7.0.
```
Comment (SenseException) [inline docs/intro/getting-started.md:5]: That would be PHP 7.1 by now.

Comment (shochdoerfer) [reply docs/intro/getting-started.md:5]: Good catch. Thanks.

**Comment by shochdoerfer** [comment]
@settermjd is this PR still needed?

**Comment by settermjd** [comment]
At this stage, since it's been so long, I'm going to scrap them and potentially look again at them.

**Pull request closed without merging**